src/scripts/orphan-client.ts

```diff
@@ -0,0 +1,175 @@
+import axios from 'axios'
+import { NdJsonTransform } from '@internal/streams/ndjson'
+import fs from 'fs'
+import path from 'path'
+
+const ADMIN_URL = ''
+const ADMIN_API_KEY = ''
```

Comment:
Can we pull these values from `process.env` instead? I'd just like to eliminate the chance of accidentally pushing a commit with secrets. Using node 20+ we can just use the `--env-file` flag so we don't need any 3rd party libraries to support this.